src/Notify/Authentication/Authenticator.cs

```diff
@@ -1,7 +1,9 @@
-﻿using Notify.Exceptions;
-using System;
+﻿using System;
 using System.Collections.Generic;
 
+using JWT;
```

Comment:
Is this needed? It wasn't present before and the code has not been updated?

Reply:
ok, can remove as I thought it might fix something